packages/message/src/DeadLetterMessage.ts

```diff
@@ -24,26 +57,25 @@ type UnwrapDeadLetterCumulusMessageReturnType = (
  * @returns {Object} the cumulus message or nearest available object
  */
 export const unwrapDeadLetterCumulusMessage = async (
-  messageBody: UnwrapDeadLetterCumulusMessageReturnType
-): Promise<UnwrapDeadLetterCumulusMessageReturnType> => {
+  messageBody: UnwrapDeadLetterCumulusMessageInputType
+): Promise<CumulusMessage | UnwrapDeadLetterCumulusMessageInputType> => {
   try {
-    if ('cumulus_meta' in messageBody) {
-      return messageBody;
-    }
-    if ('Body' in messageBody || 'body' in messageBody) {
+    if (isSQSRecordLike(messageBody)) {
       // AWS.SQS.Message/SQS.Record case
       const unwrappedMessageBody = parseSQSMessageBody(
-        messageBody
-      ) as CumulusMessage;
+        messageBody as SQSRecord | AWS.SQS.Message
+      ) as StepFunctionEventBridgeEvent;
       return await unwrapDeadLetterCumulusMessage(unwrappedMessageBody);
     }
-    if (!('detail' in messageBody)) {
```

Comment:
originally *this* is checking for the "didn't fall into any of our prepared cases" edge case
while line 45 is the "if it's an event body" case